.tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+pip install --requirement requirements.txt
```

Comment:
does `.tests.sh` assume a virtualenv has been initialised? I'm slightly worried ./.test.sh will attempt to install globally when what you really want is to just run the tests.

Reply:
Well the `README.md` instructs users to run `project_tests.sh` in which the actual tests are run via `tox` so no requirements are going to be installed globally. The reason I did not init a venv in the `.tests.sh` was that `tox` uses that script as its command.

That said, I can remove the requirements install from `.tests.sh` as we don't expect it to be run in isolation anyway. Once we have the package on PyPi then the `python setup.py install` line can be removed as well.

Reply:
sure, I get that. With the other python projects there is an assumption with 'hidden' scripts that the virtualenv has been created and is active. so running `./.test.sh` wouldn't init the venv, wouldn't install anything and would just do it's small well defined job. These hidden scripts could then be composed within a regular script that calls install or sources the venv at the top. It just makes development that little bit more convenient when you have the venv init'd in your shell and can run the lint or test or whatever directly.